### PR TITLE
Add option comment_on_pr

### LINKED ISSUE #37: Disable commenting on Pull Requests
In https://github.com/EnricoMi/publish-unit-test-result-action/issues/28#issuecomment-710080411 we have discussed that the action should allow to disable commenting on PRs. Default should be to comment on PRs as it currently does.

### PR DESCRIPTION
**Pull request by erik-f**
Resolves #37.

### CODE REVIEW

**Comment by erik-f** [comment]
I tested and commented the results in erik-f/publish-unit-test-result-action#3.

File: publish_unit_test_results.py [hunk 1]
```diff
@@ -832,6 +839,8 @@ def check_var(var: str, name: str, label: str, allowed_values: Optional[List[str
     report_individual_runs = get_var('REPORT_INDIVIDUAL_RUNS') == 'true'
     dedup_classes_by_file_name = get_var('DEDUPLICATE_CLASSES_BY_FILE_NAME') == 'true'
     hide_comment_mode = get_var('HIDE_COMMENTS') or 'all but latest'
+    # Comment on PRs if COMMENT_ON_PR is not set to 'false'
```
Comment (EnricoMi) [inline publish_unit_test_results.py:842]: I think this comment does not add much value, no extra information the code is not providing itself

File: action.yml [hunk 2]
```diff
@@ -22,6 +22,10 @@ inputs:
     description: 'Configures which earlier comments of the action to hide: off - no comments, orphaned commits - comments for commits that are removed from the branch, all but latest - all comments but the latest'
     required: false
     default: 'all but latest'
+  comment_on_pr:
+    description: 'Toggle PR comments: false - disable PR comments, true - comment on PRs'
```
Comment (EnricoMi) [inline action.yml:26]: Maybe: `Control PR comments: ...`

**Review by EnricoMi — CHANGES_REQUESTED** [review]
Thanks for the PR, looks good. Only minor change requests.

File: publish_unit_test_results.py (hunk 1 above)
Comment (erik-f) [reply publish_unit_test_results.py:842]: I added it after I messed up this line yesterday. Looking at it now, I think it's quite self-explanatory.

**Review by EnricoMi — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**